config/MezaCoreExtensions.yml

```diff
@@ -530,8 +530,8 @@ list:
 
   # HeaderFooter - Provides per-namespace and per-page header and footer inclusion.
   - name: HeaderFooter
-    repo: https://github.com/enterprisemediawiki/HeaderFooter.git
-    version: tags/3.0.1
+    repo: https://github.com/wikimedia/mediawiki-extensions-HeaderFooter.git
+    version: "{{ mediawiki_default_branch }}"
```

Comment:
The default branch for the github repo is REL1_39_Compat, but REL1_39 is the correct branch to use because it contains the Compat code, **and** automatically receives translatewiki updates because it's a REL branch.